bin/update-pr-environment

```diff
@@ -35,6 +35,23 @@ service_name="$(terraform -chdir="infra/$app_name/service" output -raw service_n
 echo "Wait for service ${service_name} to become stable"
 aws ecs wait services-stable --cluster "${cluster_name}" --services "${service_name}"
 
-echo "Post comment to PR with service endpoint"
 service_endpoint="$(terraform -chdir="infra/${app_name}/service" output -raw service_endpoint)"
-gh pr comment "${pr_number}" -b "Updated PR environment at ${service_endpoint}"
+pr_info=$(cat <<EOF
+<!-- begin PR environment info -->
+## Preview environment
+Service endpoint: ${service_endpoint}
+Deployed commit: ${image_tag}
```

Comment:
Nit: I think this would be more legible if these were bullets. I don't know if github will render them correctly, but possible worth a try in your `platform-test` PR.
```suggestion
- Service endpoint: ${service_endpoint}
- Deployed commit: ${image_tag}
```